app/views/layouts/application.html.erb

```diff
@@ -83,13 +83,18 @@
         </ul>
       </h4>
     </header>
+    
+    <% flash.each do |type, message| %>
```

Comment:
We don't need this since it this block of code was added in my last PR. When I get the merge conflicts, I will remove this.